src/content/docs/granted/recipes/assume-in-scripts.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: "Assume Roles in Commands"
+---
+
+## overview
+
+Normally we are prompted to choose a role when using assume however there are options available which make it possible to build scripts that assume without prompt (unless your token has expired)
+
+This workflow/recipe is suitable for people who need to perform attended scripts using their SSO.
+
+It can also be adapted to script across multiple accounts in your org which is otherwise relatively tedeous.
+
+This solution builds upon the basic concept of the inline-role-assumption
```

Comment:
I've opted to remove this for brevity - we can add it back in, but we might want to add a hyperlink on `inline-role-assumption` to link somewhere that explains what it is, in case the reader hasn't heard of that before.

```suggestion
```